alpine-ruby/Dockerfile

```diff
@@ -29,9 +29,9 @@ RUN echo 'gem: --no-document' > ~/.gemrc \
 
 # Setup bundler for the root user
 RUN bundle config --global -- build.nokogiri '--use-system-libraries' \
-  && bundle config --global -- frozen 'true' \
+  && bundle config --global -- frozen '1' \
```

Comment:
Technically `bundler` treats them the same, but when you use `bundle --deployment` it sets this to `'1'` under the hood, so I thought it was important to stick to those semantics.